docs/AVS-Guide.md

```diff
@@ -0,0 +1,85 @@
+    - A “DataStore” in the context of EigenDA
+    - Posting a state root of another blockchain for a bridge service
+
+2. *Stake is “At Stake” on Tasks for a Finite Duration*: 
+    It is assumed that every task (eventually) resolves. Each operator places their stake in EigenLayer “at stake” on the tasks that they perform. In order to “release” the stake (e.g. so the operator can withdraw their funds), these tasks need to eventually resolve. It is recommended, but not required that a predefined duration is specified in the AVS contract for each task. As a guideline, the EigenLabs team believes that the duration of a task should be aligned with longest reasonable duration that would be acceptable for an operator to keep funds “at stake”. An AVS builder should recognize that extending the duration of a task may impose significant negative externalities on the stakers of EigenLayer, and may disincentivize stakers opting-in to serving their application.
+
+3. *Services Slash Only Objectively Attributable Behavior*: 
+    EigenLayer is built to support slashing as a result of an on-chain-checkable, objectively attributable action. An AVS SHOULD slash in EigenLayer only for such provable and attributable behavior. It is expected that operators will be very hesitant to opt-in to services that slash for other types of behavior, and other services may even choose to exclude operators who have opted-into serving one or more AVSs with such “subjective slashing conditions”, as these slashing conditions present a significant challenge for risk modeling, and may be perceived as more dangerous in general. Some examples of on-chain-checkable, objectively attributable behavior: 
+    - double-signing a block in Ethereum, but NOT inactivity leak; 
+    - proofs-of-custody in EigenDA, but NOT a node ceasing to serve data; 
+    - a node in a light-node-bridge AVS signing an invalid block from another chain.
+
+4. *Single Point-of-Interaction for Services and EigenLayer*: 
+    It is assumed that services have a single contract that coordinates the service’s communications sent to EigenLayer. This contract – referred to as the ServiceManager – informs EigenLayer of operator registration, updates, and deregistration, as well as signaling to EigenLayer when an operator should be slashed (frozen). An AVS has full control over how it splits the actual logic involved, but is expected to route all calls to EigenLayer through a single contract. While technically possible, an AVS SHOULD NOT use multiple contracts to interact with EigenLayer due to a couple of reasons. Firstly, multiple contracts will impose additional burden on stakers in EigenLayer to withdrawing stake. Secondly, operators will be faced with the complexity of selecting the correct contract in order to deregister from the AVS.
```

Comment:
suggestion: delete "due to a couple of reasons. Firstly, multiple contracts"
replace with: ". An AVS architecture using multiple contracts to interact with EigenLayer"

also: "to withdrawing stake" => "when withdrawing stake"

also delete "Secondly, operators will be faced with the complexity of selecting the correct contract in order to deregister from the AVS." as I don't think this makes sense.

The general problem here is that interactions would have to be repeated for all of an AVS's contracts that communicate with EigenLayer -- so having only one contract is much simpler and minimizes this "burden"